config/ceph-config.service

```diff
@@ -0,0 +1,35 @@
+[Unit]
```

Comment:
This is consistent with the other components.  It's just an example systemd unit file.  I think they _should_ be included, as they serve as good, practical documentation.

It's not useful to include them in the Docker images, of course, but as a part of the Github repo, I think they are invaluable.